src/EditorFeatures/VisualBasicTest/AddAccessibilityModifiers/AddAccessibilityModifiersTests.vb

```diff
@@ -202,5 +212,195 @@ namespace N
     end namespace
 end namespace")
         End Function
+
+                <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddAccessibilityModifiers)>
+        Public Async Function TestAllConstructsWithOmit() As Task
+            Await TestInRegularAndScriptAsync(
+"
+namespace N
+    namespace Outer.Inner
+        Friend class {|FixAllInDocument:C|}
+            Public class NestedClass
+            end class
+
+            Public structure NestedStruct
+            end structure
+
+            Private f1 as integer
+            Private f2, f3 as integer
+            Private f4, f5 as integer, f6, f7 as boolean
+            public f4 as integer
+
+            Public event e1 as Action
+            public event e2 as Action
+
+            Public custom event e4 as Action
+            end event
+
+            shared sub new()
+            end sub
+
+            Public sub new()
+            end sub
+
+            public sub new(i as integer)
+            end sub
+
+            Public sub M1()
+            end sub
+
+            public sub M2()
+            end sub
+
+            Public function M3() as integer
+            end function
+
+            Public function M4() as integer
+
+            public function M5() as integer
+            end function
+
+            Private partial sub M6()
+            end sub
+
+            Public property P1 as integer
+
+            Public property P2 as integer
+                get
+                end get
+            end property
+
+            public property P3 as integer
+
+            Public shared operator &(c1 as C, c2 as C) as integer
+            end operator
+        end class
+
+        Friend interface I
+            event e6 as Action
+            sub M3()
+            function M4() as integer
+            property P3 as integer
+        end interface
+
+        Friend delegate sub D1()
+        Friend delegate function D2() as integer
+
+        Friend enum E
+            EMember
+        end enum
+
+        Friend structure S
+            Public f as integer
+
+            Public sub M()
+            end sub()
+
+            Public shared operator &(c1 as S, c2 as S) as integer
+            end operator
+        end structure
+
+        Friend module M
+            Private f as integer
+
+            Public sub M()
+            end sub()
```

Comment:
is there a reason sub has () at the end of it?

Reply:
oh, you copied from above.  we should probably remove this from all of them :)

Reply:
There's other wonky/invalid syntax. Part of me wonders if that's intentional to make sure the fixes are tolerant of that?

Reply:
> Part of me wonders if that's intentional

it was not intentional.  I did a quick/dirty copy/paste of the C# code into the VB string and just hacked at it there :)